cfme/tests/containers/test_basic_metrics.py

```diff
@@ -1,19 +1,17 @@
 import pytest
+import re
 from utils import testgen
 from utils import conf
 from utils.version import current_version
 
 
 pytestmark = [
     pytest.mark.uncollectif(lambda provider: current_version() < "5.6"),
-    pytest.mark.usefixtures('setup_provider'),
```

Comment:
Yes, no idea why it was deleted from the test. 
The purpose of the test is very simple, to check whether the OpenShift provider we work against has functioning metrics (We have this recurring issue and would like to detect it as a part of our tests run)